cron/reminder_notice.pl

```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/perl
+
+use Modern::Perl;
+use Encode;
+use DateTime;
+use YAML::Syck;
+use Data::Dumper;
+use utf8;
+$| = 1; # Don't buffer output
+
+use Koha::Checkouts;
+use Koha::DateUtils qw( dt_from_string );
+use Koha::Illrequests;
+use Koha::Illrequestattributes;
+use Koha::Illbackends::Libris::Base;
+use Koha::Items;
+use Koha::Notice::Messages;
+use Koha::Patrons;
+use C4::Context;
+use C4::Letters;
+binmode(STDOUT, ":utf8");
+my $dbh = C4::Context->dbh;
+
+# Get the path to, and read in, the Libris ILL config file
+my $ill_config_file = C4::Context->config('interlibrary_loans')->{'libris_config'};
+my $ill_config = LoadFile( $ill_config_file );
+my $reminders = $ill_config->{ 'reminders' };
+
+# find received inter library loans
+my $receiveds = Koha::Illrequests->search({
+    status => 'IN_ANK'
+});
+
+my $now = DateTime->now;
+
+foreach my $received ( $receiveds->as_list ) {
+    # check if reminder configured
+    my $attributes = Koha::Illrequestattributes->search({
+        illrequest_id => $received->illrequest_id
+    });
+    next unless $attributes;
```

Comment:
Looks like $attributes is not used anywhere?

Reply:
Removed in cf7e9ee